recipe/meta.yaml

```diff
@@ -74,17 +74,20 @@ requirements:
     - jpeg
     - libtiff
     - libpng
+    - openexr
+    - lemon
     - fftw
     - hdf5
     - boost
     - zlib
-    
+
   run:
     - python
     - {{ pin_compatible('numpy') }}
     - jpeg
     - libtiff
     - libpng
+    - lemon
```

Comment:
FWIW `lemon` only has a static library. So we can probably drop it from `run`.

Reply:
ah ok. thx.

Reply:
And I think we need to add `openexr` to the `run` requirements.  Looks like it was accidentally dropped when `lemon` was added.

Geez, I failed as a reviewer here... sorry guys.

Reply:
No, that one has a run exports
https://github.com/conda-forge/openexr-feedstock/blob/master/recipe/meta.yaml#L18

Reply:
oh ok.  Sorry for the noise, then.

Reply:
not an issue

Reply:
It's all good 🙂